egs/ami/s5b/RESULTS_ihm

```diff
@@ -55,17 +61,21 @@
 %WER 22.4 | 12643 89977 | 80.3 12.5 7.2 2.7 22.4 53.6 | -0.503 | exp/ihm/nnet3_cleaned/lstm_bidirectional_sp/decode_eval/ascore_10/eval_hires.ctm.filt.sys
 
 ############################################
-
-# local/chain/run_tdnn.sh --mic ihm --stage 12 &
 # cleanup + chain TDNN model
-# for d in exp/ihm/chain_cleaned/tdnn_sp_bi/decode_*; do grep Sum $d/*sc*/*ys | utils/best_wer.sh; done
+# local/chain/run_tdnn.sh --mic ihm --stage 12 &
+# for d in exp/ihm/chain_cleaned/tdnn_sp_bi/decode_*; do grep sum $d/*sc*/*ys | utils/best_wer.sh; done
 %WER 22.5 | 13098 94490 | 80.6 10.8 8.6 3.1 22.5 55.0 | 0.072 | exp/ihm/chain_cleaned/tdnn_sp_bi/decode_dev/ascore_10/dev_hires.ctm.filt.sys
 %WER 22.5 | 12643 89978 | 80.3 12.5 7.2 2.7 22.5 53.1 | 0.149 | exp/ihm/chain_cleaned/tdnn_sp_bi/decode_eval/ascore_10/eval_hires.ctm.filt.sys
 
+# cleanup + chain TDNN model. Uses PCA instead of LDA for ivector features.
```

Comment:
Could you rerun the baseline?  I suspect something else may have changed since it was last run.
And can you put the output of one of those compare_wer.sh scripts inside the 1d script, showing
how it differs from 1b?
If 1d is the best script so far, you could update the link.